schema/sale_artwork.js

```diff
@@ -215,16 +218,27 @@ const SaleArtworkType = new GraphQLObjectType({
         type: GraphQLInt,
         deprecationReason: 'Favor `counts.bidder_positions`',
       },
-      bid_increment: {
-        type: GraphQLInt,
+      bid_increments: {
+        type: new GraphQLList(GraphQLInt),
         resolve: ({ minimum_next_bid_cents }) => {
           return gravity('/increments').then((res) => {
             const deflt = find(res, { key: 'default' });
-            const bucket = find(deflt.increments, (inc) =>
-              minimum_next_bid_cents >= inc.from &&
-              minimum_next_bid_cents <= inc.to
-            );
-            return bucket.amount;
+            const increments = [minimum_next_bid_cents];
```

Comment:
Or, embrace mutability and do a generator:

```
function* increments(from, to) {
  let tiers = deflt.increments.slice();
  for (let next = from; next <= to; next += tiers[0].amount) {
    if (next > tiers[0].to && tiers.length > 0) tiers = tiers.slice(1);
    yield next;
  }
}
```

Then you can do `[...increments(1000, 500000)]`